google-cloud-examples/src/main/java/com/google/cloud/examples/bigquery/snippets/JobSnippets.java

```diff
@@ -89,10 +104,55 @@ public boolean waitForWithOptions() throws InterruptedException {
       } else {
         // job completed successfully
       }
-      // [END waitFor]
+      // [END waitForWithOptions]
     } catch (TimeoutException e) {
       return true;
     }
     return true;
   }
+
+  /**
+   * Example of reloading all fields until job status is DONE.
+   */
+  // [TARGET reload(JobOption...)]
+  public JobStatus.State reload() throws InterruptedException {
+    Job job = this.job;
```

Comment:
Uh sorry, the same name confused me. Actually I prefer as it was, but feel free to also merge as is.